retreaver-typescript/src/Campaign.ts

```
@@ -0,0 +1,398 @@
+   * Handle ClickFlare integration
+   * @param number - Number instance
+   */
+  private handleClickFlareIntegration(number: RetreaverNumber): void {
+    const clickFlareConfig: ClickFlareConfig = this.getIntegrationConfig(number, "click_flare");
+
+    if (typeof clickFlareConfig === "undefined") {
+      return;
+    }
+
+    const getClickFlareClickID = (): string | undefined => {
+      if ((window as any).clickflare && (window as any).clickflare.data && (window as any).clickflare.data.event_tokens) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)